src/main/java/com/vikingzorros/rehabit/daorepositories/PostRepository.java

```diff
@@ -10,4 +10,5 @@
         List<Post> findByCategory_id(int theId);
        //  List<Post> findByCategory_idOrderByCreate_timeDesc(int theId);
 
+
```

Comment:
Method name is not according to the java method naming conventions

Reply:
I have changed that in last commit.